internal/util/metrics/metrics.go

```diff
@@ -25,50 +25,160 @@ import (
-var CacheRequestElements *prometheus.CounterVec
+// ProxyRequestElements ...
+var ProxyRequestElements *prometheus.CounterVec
 
 // ProxyRequestDuration ...
 var ProxyRequestDuration *prometheus.HistogramVec
 
-// Init ...
+// CacheObjectOperations ...
+var CacheObjectOperations *prometheus.CounterVec
+
+// CacheByteOperations ...
+var CacheByteOperations *prometheus.CounterVec
+
+// CacheEvents ...
+var CacheEvents *prometheus.CounterVec
+
+// CacheObjects ...
+var CacheObjects *prometheus.GaugeVec
+
+// CacheBytes ...
+var CacheBytes *prometheus.GaugeVec
+
+// CacheMaxObjects ...
+var CacheMaxObjects *prometheus.GaugeVec
+
+// CacheMaxBytes ...
+var CacheMaxBytes *prometheus.GaugeVec
+
+// Init initializes the instrumented metrics and starts the listener endpoint
 func Init() {
 
 	ProxyRequestStatus = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
-			Name: "trickster_requests_total",
-			Help: "Count of ",
+			Namespace: metricNamespace,
+			Subsystem: proxySubsystem,
+			Name:      "requests_total",
+			Help:      "Count of downstream client requests handled by trickster",
 		},
-		[]string{"origin", "origin_type", "method", "cache_status", "http_status", "path"},
+		[]string{"origin_name", "origin_type", "method", "cache_status", "http_status", "path"},
 	)
 
-	CacheRequestElements = prometheus.NewCounterVec(
+	ProxyRequestElements = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
-			Name: "trickster_points_total",
-			Help: "Count of data points returned in Upstream Timeseries Requests",
+			Namespace: metricNamespace,
+			Subsystem: proxySubsystem,
+			Name:      "points_total",
+			Help:      "Count of data points in the timeseries returned to the requesting client.",
 		},
-		[]string{"origin", "origin_type", "cache_status", "path"},
+		[]string{"origin_name", "origin_type", "cache_status", "path"},
 	)
 
 	ProxyRequestDuration = prometheus.NewHistogramVec(
 		prometheus.HistogramOpts{
-			Name:    "trickster_proxy_duration_seconds",
-			Help:    "Time required in seconds to proxy a given Prometheus query.",
-			Buckets: []float64{0.05, 0.1, 0.5, 1, 5, 10, 20},
+			Namespace: metricNamespace,
+			Subsystem: proxySubsystem,
+			Name:      "request_duration_seconds",
+			Help:      "Time required in seconds to proxy a given Prometheus query.",
+			Buckets:   []float64{0.05, 0.1, 0.5, 1, 5, 10, 20},
+		},
+		[]string{"origin_name", "origin_type", "method", "status", "http_status", "path"},
+	)
+
+	CacheObjectOperations = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: metricNamespace,
+			Subsystem: cacheSubsystem,
+			Name:      "object_operations_total",
+			Help:      "Count (in # of objects) of operations performed on a Trickster cache.",
+		},
+		[]string{"cache_name", "cache_type", "operation", "status"},
+	)
+
+	CacheByteOperations = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: metricNamespace,
+			Subsystem: cacheSubsystem,
+			Name:      "byte_operations_total",
```

Comment:
If this is a byte counter, it should be `operation_bytes_total`. This makes it sound like it's a counter of operations.